### DISCUSSION TITLE
Kubernetes Cluster RKE2 - Evicted Pod

### DISCUSSION THREAD

**Discussion by lucasdantasuni** [community/community#84978] [Programming Help] [answered]
### Body

Estou tendo um problema em meu cluster kubernetes no qual em certos momentos meu pod fica com o estado de "evicted" como posso fazer para resolver? 

![e1b7f0d7-dde7-4ece-8c0b-c6c718b4fe9c_img](https://github.com/community/community/assets/155675604/626b20df-63ec-4d00-b206-25a75549e960)

Estou usando o RKE2 em um cluster on-premise; 

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by LuksJobs** [comment, marked as answer]
Opa, beleza!? A resolução de pods em estado de “evicted” depende da causa subjacente do problema. Aqui estão algumas possíveis soluções técnicas para resolver esse problema:

Verificar os logs e eventos do Kubernetes: O primeiro passo para solucionar um pod em estado de “evicted” é verificar os logs e eventos do Kubernetes para determinar a causa subjacente do problema. Isso pode ajudar a identificar problemas de rede, falta de recursos ou outros problemas que possam estar afetando o pod.

Para listar os pods de um namespace especifico que está em estado de evicted, você pode utilizar o comando abaixo:

`kubectl get pods --all-namespaces --field-selector=status.phase=Failed | grep Evicted`

Em geral, resolver pods em estado de “evicted” pode ser um processo complexo que exige uma análise cuidadosa do ambiente Kubernetes e da configuração do pod. Certifique-se de investigar cuidadosamente as causas subjacentes do problema antes de tentar uma solução.
